recipes/scalapack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "scalapack" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "0c74aeae690fe5ee4db7926f49c5d0bb69ce09eea75beb915e00bba07530395c" %}
+{% set mpi = 'mpich' %}
+{% set openblas_version = '0.2.19' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.netlib.org/{{ name }}/{{ name }}-{{ version }}.tgz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ mpi }}
+    - cmake
+    - gcc
+    - openblas {{openblas_version}}
+  run:
+    - {{ mpi }}
+    - openblas {{openblas_version}}
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libscalapack.dylib  # [osx]
+    - test -f $PREFIX/lib/libscalapack.so  # [linux]
```

Comment:
Only suggestion I have is to add this to the test section:
```yaml
    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```
but that can be done in the feedstock too. Let's just get this in. Sorry for the long delay on merging this recipe.